README.org

```diff
@@ -117,6 +117,38 @@ In order to create a hydra, bind it to a key using =ryo-modal-key= or =ryo-modal
                ("q" nil "cancel" :color blue)))
 #+END_SRC
 
+** Use-package keyword
+
+Ryo-modal also provides a use-package keyword: =:ryo-modal=, which is similar to =:bind= in that it
+implies =:defer t= and create autoloads for the bound commands. The keyword is followed by one or
+more key-binding commands, using the same syntax as used by =ryo-modal-keys= as is illustrated by
+the following example:
+
+#+begin_src emacs-lisp
+(use-package modal-test
+  :ensure nil
+  :ryo-modal
+  ;; A list of keywords will be applied to all following keybindings up to the next list of keywords.
+  (:mode 'org-mode :norepeat t)
+  ("0" "M-0")
+  ("G" end-of-buffer :name "insert at buffer end" :read t)
+
+  ;; This new list of keywords will reset the applied defaults; it applies to all keybindings following.
+  (:norepeat t)
+  ("SPC" (("n" next-line :name "my next line")
+          ("p" previous-line)))
+  ;; NOTE: default keywords are currently ignored when using :hydra
```

Comment:
Could you please try if keywords work now, with the latest commit? My guess is that they should.

Reply:
Now tested with the latest commit. The keywords now get applied correctly when using :hydra. That is really nice!!